pkg/image/api/v1/conversion.go

```diff
@@ -97,13 +95,13 @@ func convert_v1_Image_To_api_Image(in *Image, out *newer.Image, s conversion.Sco
 	return nil
 }
 
-func convert_v1_ImageStreamSpec_To_api_ImageStreamSpec(in *ImageStreamSpec, out *newer.ImageStreamSpec, s conversion.Scope) error {
+func Convert_v1_ImageStreamSpec_To_api_ImageStreamSpec(in *ImageStreamSpec, out *newer.ImageStreamSpec, s conversion.Scope) error {
 	out.DockerImageRepository = in.DockerImageRepository
 	out.Tags = make(map[string]newer.TagReference)
 	return s.Convert(&in.Tags, &out.Tags, 0)
 }
 
-func convert_api_ImageStreamSpec_To_v1_ImageStreamSpec(in *newer.ImageStreamSpec, out *ImageStreamSpec, s conversion.Scope) error {
+func Convert_api_ImageStreamSpec_To_v1_ImageStreamSpec(in *newer.ImageStreamSpec, out *ImageStreamSpec, s conversion.Scope) error {
```

Comment:
This is the only method in image that has defaulting

On Tue, Mar 8, 2016 at 11:47 AM, David Eads <notifications@github.com>
wrote:

> In pkg/image/api/v1/conversion.go
> <https://github.com/openshift/origin/pull/7814#discussion_r55397002>:
>
> >  	out.DockerImageRepository = in.DockerImageRepository
> >  	out.Tags = make(map[string]newer.TagReference)
> >  	return s.Convert(&in.Tags, &out.Tags, 0)
> >  }
> >
> > -func convert_api_ImageStreamSpec_To_v1_ImageStreamSpec(in *newer.ImageStreamSpec, out *ImageStreamSpec, s conversion.Scope) error {
> > +func Convert_api_ImageStreamSpec_To_v1_ImageStreamSpec(in *newer.ImageStreamSpec, out *ImageStreamSpec, s conversion.Scope) error {
>
> This is from internal to external, so it doesn't need a conversion. The
> one above needs to call the defaults, which is weird because I thought I
> changed it.
>
> It's a problem throughout the pull.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7814/files#r55397002>.
>